pkg/apis/sources/v1alpha1/containersource_types.go

```diff
@@ -24,6 +24,9 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// Important: Run "make" to regenerate code after modifying this file
```

Comment:
this comment is no longer valid, we have no make file.

Reply:
Comment removed